vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -84,6 +95,21 @@ switch ($windowsSKU) {
             "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod10132021",
             "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod01312022")
     }
+    "2022-containerd" {
+        $global:imagesToPull = @(
+            "mcr.microsoft.com/windows/servercore:ltsc2022",
+            "mcr.microsoft.com/windows/nanoserver:ltsc2022",
+            "mcr.microsoft.com/oss/kubernetes/pause:3.6-hotfix.20220114",
+            "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.5.0",
+            "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.4.0",
+            "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.10.0",
+            "mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:v1.9.0",
+            "mcr.microsoft.com/oss/azure/secrets-store/provider-azure:v1.0.0",
+            "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:v1.0.0",
+            "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.23.1", # for k8s 1.23.x
+            "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod10132021",
```

Comment:
I think that you can remove ciprod:win-ciprod10132021 for 2022-containerd since this is an old image. BTW: We also can use it for 2019 since AKS uses win-ciprod01312022 now.

Reply:
Updated. Thank you.